frontend-ssr/pages/api/notifications/mockAllAsRead.ts

```diff
@@ -0,0 +1,57 @@
+import { uuid } from '~/services/Utils';
```

Comment:
We shouldn't need this file. We don't return anything when marking all notifications as read.